package.json

```diff
@@ -60,6 +60,7 @@
     "@types/chai": "^4.2.3",
     "@types/expect": "^1.20.4",
     "@types/mocha": "^5.2.7",
+    "@types/node-fetch": "^2.5.2",
     "@types/request": "^2.48.3",
```

Comment:
We should remove this as request is not being used anymore.

Reply:
Sorry, but we started re-using the types of request in a method that translates requestOptions to fetchOptions